backend/core/src/email/email.service.ts

```diff
@@ -224,34 +255,115 @@ export class EmailService {
+          summaries.minIncome.push(Number.parseFloat(unit.monthlyIncomeMin))
+        }
+        if (unit.annualIncomeMax) {
+          summaries.maxIncome.push(Number.parseFloat(unit.annualIncomeMax) / 12.0)
+        }
+        if (unit.monthlyRent) {
+          summaries.rent.push(Number.parseFloat(unit.monthlyRent))
+        }
+        const thisBedroomInfo = summaries.bedrooms[unit.unitType?.name]
+        summaries.bedrooms[unit.unitType?.name] = thisBedroomInfo
+          ? [...thisBedroomInfo, unit]
+          : [unit]
+        return summaries
+      },
+      {
+        bedrooms: {},
+        rent: [],
+        minIncome: [],
+        maxIncome: [],
+      }
+    )
+    const tableRows = []
+    if (listing.reservedCommunityType) {
+      tableRows.push({
+        label: this.polyglot.t("rentalOpportunity.community"),
+        value: formatCommunityType[listing.reservedCommunityType?.name],
+      })
+    }
+    if (listing.applicationDueDate) {
+      tableRows.push({
+        label: this.polyglot.t("rentalOpportunity.applicationsDue"),
+        value: dayjs(listing.applicationDueDate).format("MMMM D, YYYY"),
+      })
+    }
+    tableRows.push({
+      label: this.polyglot.t("rentalOpportunity.address"),
+      value: `${listing.buildingAddress.street}, ${listing.buildingAddress.city} ${listing.buildingAddress.state} ${listing.buildingAddress.zipCode}`,
+    })
+    Object.entries(units.bedrooms).forEach(([key, bedroom]) => {
+      const sqFtString = formatUnitDetails(bedroom, "sqFeet", "sqft")
+      const bathroomstring = formatUnitDetails(bedroom, "numBathrooms", "bath", "baths")
+      tableRows.push({
+        label: this.polyglot.t(`rentalOpportunity.${key}`),
+        value: `${bedroom.length} unit${
+          bedroom.length > 1 ? "s" : ""
+        }${bathroomstring}${sqFtString}`,
+      })
+    })
+    if (units.rent?.length) {
+      tableRows.push({
+        label: this.polyglot.t("rentalOpportunity.rent"),
+        value: formatPricing(units.rent),
+      })
+    }
+    if (units.minIncome?.length) {
+      tableRows.push({
+        label: this.polyglot.t("rentalOpportunity.minIncome"),
+        value: formatPricing(units.minIncome),
+      })
+    }
+    if (units.maxIncome?.length) {
+      tableRows.push({
+        label: this.polyglot.t("rentalOpportunity.maxIncome"),
+        value: formatPricing(units.maxIncome),
+      })
+    }
+    if (listing.events && listing.events.length > 0) {
+      const lotteryEvent = listing.events.find(
+        (event) => event.type === ListingEventType.publicLottery
+      )
+      if (lotteryEvent && lotteryEvent.startDate) {
+        tableRows.push({
+          label: this.polyglot.t("rentalOpportunity.lottery"),
+          value: dayjs(lotteryEvent.startDate).format("MMMM D, YYYY"),
+        })
+      }
+    }
+
+    const compiled = compiledTemplate({
+      listingName: listing.name,
+      listingUrl: `${jurisdiction.publicUrl}/listings/${listing.id}`,
```

Comment:
I believe this should just be "listing" since currently getting a page not found with listings

Reply:
Nice catch! I updated this